_examples/objectstore/main.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 Fastly, Inc.
+
+package main
```

Comment:
Non-Blocking Comment:
It's great we have an example app and build it.

Should we also use these for end-to-end testing of the SDK? We do that for the JS SDK and it has helped increase confidence in our implementation and also caught differences between Viceroy and C@E